src/PrestaShopBundle/Controller/Admin/Sell/Order/ActionsBarButton.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace PrestaShopBundle\Controller\Admin\Sell\Order;
```

Comment:
```suggestion
 */

declare(strict_types=1):

namespace PrestaShopBundle\Controller\Admin\Sell\Order;
```

Reply:
I hesitated about that.
Because the `PrestaShop\PrestaShop\Core\ActionBar\ActionsBarButton` do not have strict types, I preferred to not put it here

Reply:
Agreed if they were not present in this initla class in 178x branch then it makes sense not to reinsert it here.
However you can add them in the new classes in the `PrestaShop\PrestaShop\Core\ActionBar`namespace